index.js

```diff
@@ -9,6 +11,7 @@ const memo = async(key, fn, ttl, forceUpdate) => {
     }
   }
   const result = await fn();
+  timestamps[key] = new Date().getTime();
```

Comment:
could you just set the header here? and then not have to worry about the timetamps object